common/json.go

```diff
@@ -187,3 +188,12 @@ func addAcceptJSONHeader(headers []Header) []Header {
 
 	return append(headers, Header{Key: "Accept", Value: "application/json"})
 }
+
+// AddJSONSuffix adds the suffix ".json" to an objectName.
+func AddJSONSuffix(objectName string) string {
```

Comment:
If we want to have this in `common`, let's repurpose this to be more generic - `AddSuffixIfNotExists(str string, suffix str)`